packages/esm-service-queues-app/src/metrics/metrics-cards/waiting-patients.extension.tsx

```diff
@@ -34,9 +34,9 @@ export default function WaitingPatientsExtension() {
     service: currentService?.serviceUuid,
     location: currentQueueLocation,
     isEnded: false,
+    status: 'WAITING',
```

Comment:
Should we use the `waiting` status concept UUID here for more robustness?

```tsx
  const config = useConfig<ConfigObject>();
  const waitingStatusUuid = config?.concepts?.defaultStatusConceptUuid;
  const { serviceCount } = useServiceMetricsCount(
    currentService?.serviceUuid,
    currentQueueLocation,
    waitingStatusUuid,
  );
```

And then:

```tsx
    status: waitingStatusUuid,
```